src/vs/workbench/contrib/terminalContrib/clipboard/browser/terminalClipboard.ts

```diff
@@ -9,14 +9,84 @@ import { IDialogService } from '../../../../../platform/dialogs/common/dialogs.j
 import { ServicesAccessor } from '../../../../../platform/instantiation/common/instantiation.js';
 import { TerminalSettingId } from '../../../../../platform/terminal/common/terminal.js';
 
-export async function shouldPasteTerminalText(accessor: ServicesAccessor, text: string, bracketedPasteMode: boolean | undefined): Promise<boolean | { modifiedText: string }> {
+class SmartPasteUtils {
+	/**
+	 *
+	 * @param string
+	 * @returns true if the string is a path
+	 */
+	static isPath(string: string): boolean {
```

Comment:
We don't want to use keywords like `string` as variable names, `text` is a good alternative.